pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```
@@ -83,6 +83,26 @@ podLogs:
         {{- range .DefaultWorkloadClusterNamespaces }}
         - {{ . }}
         {{- end }}
+    relabelings:
+    - action: replace
+      targetLabel: "giantswarm_observability_tenant"
+      replacement: {{ .DefaultWriteTenant }}
+- name: customers-logs
+  namespace: kube-system
+  spec:
+    selector: {}
```

I see we are selecting every PodLogs here, I am worried we run into cases where customers wants to run their own logging pipeline and suddenly they get their logs going through our infra as a side unwanted effect.
Using an explicit label to configure those PodLogs to selected by Giantswarm infrastructure would make it safer and more declarative.

---

What do you think about this then?

```suggestion
    selector:
      matchExpressions:
      - key: observability.giantswarm.io/tenant
        operator: Exists
```

---

That works